cellprofiler/modules/filterobjects.py

```diff
@@ -296,17 +299,18 @@ def set_directory_fn(path):
             self.rules_directory.join_parts(dir_choice, custom_path)
 
         self.rules_file_name = cellprofiler.setting.FilenameText(
-            "Rules file name",
+            "Rules or classifier file name",
             "rules.txt",
             get_directory_fn=get_directory_fn,
             set_directory_fn=set_directory_fn,
             doc="""\
 *(Used only when filtering using {MODE_RULES} or {MODE_CLASSIFIERS})*
 
-The name of the rules file. This file should be a plain text file
-containing the complete set of rules.
+The name of the rules or classifier file. 
+
+A rules file is a plain text file containing the complete set of rules.
 
-Each line of this file should be a rule naming a measurement to be made
+Each line of the rules file should be a rule naming a measurement to be made
```

Comment:
Changes look great throughout... this was the part I was most uncertain about. Do all files from Classifier definitely have this structure? I would think those produced using GentleBoosting (the original default classifier type) would, but I'm not sure if you can now use SVM and other options from CPA's Classifier in CP here... and if so, are those types accurately described by this wording? I suspect not in which case I think we can readily resolve by just saying, "For example, if you chose GentleBoosting as the classifier in CellProfiler Analyst, each line of the file....". 

I can't rule out that my questions even make sense, because I don't have full understanding of this module!

For this line specifically, I think we probably don't want to say "the rules", wouldn't it be "the rules or classifier" (or just leave as is)?

Reply:
Keep reading! :D

Reply:
bwaaa haaa haaaa